### PR TITLE
Fixes

### PR DESCRIPTION
**Pull request by zBritva**

### CODE REVIEW

**Comment by zBritva** [comment]
@danmarshall Could you review the PR?

File: src/app/utils/index.ts [hunk 1]
```diff
@@ -233,7 +233,7 @@ export function getConvertableDataKind(
       types = [DataKind.Categorical, DataKind.Ordinal];
       break;
     case DataType.Number:
-      types = [DataKind.Categorical, DataKind.Numerical, DataKind.Ordinal];
+      types = [DataKind.Categorical, DataKind.Numerical];
```
Comment (danmarshall) [inline src/app/utils/index.ts:236]: Hmmm I think number can still be ordinal ?

File: src/app/utils/index.ts [hunk 2]
```diff
@@ -352,3 +352,19 @@ export function isInIFrame() {
...
+export function getAligntment(anchor: Element) {
+  let alignX:
+    | "start-outer"
+    | "inner"
+    | "outer"
+    | "start-inner"
+    | "end-inner"
+    | "end-outer";
+  let alignLeft: boolean = false;
+  if (window.innerWidth - anchor.getBoundingClientRect().x < 500) {
```
Comment (danmarshall) [inline src/app/utils/index.ts:365]: can you make a constant for the 500 or explain with a comment

File: src/app/views/menubar.tsx [hunk 3]
```diff
@@ -445,6 +447,96 @@ export class MenuBar extends ContextedComponent<
...
+                    <div
+                      onMouseDown={(e) => {
+                        e.stopPropagation();
+                      }}
+                      className={"charticulator__reset_chart_dialog"}
+                    >
+                      <div
+                        className={"charticulator__reset_chart_dialog-inner"}
+                      >
+                        {/* <ModalView context={context}> */}
```
Comment (danmarshall) [inline src/app/views/menubar.tsx:497]: remove

**Review by danmarshall — APPROVED** [review]

**Comment by danmarshall** [comment]
Approved, just a couple of minor comments.

File: src/app/utils/index.ts (hunk 1 above)
Comment (zBritva) [reply src/app/utils/index.ts:236]: Agree. But we decided to exclude for numerical in the last sync meeting. @bongshin could you confirm?

**Pull request merged**